nvflare/app_common/filters/params_converter_filter.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Union
+
+from nvflare.apis.dxo_filter import DXO, DXOFilter
+from nvflare.apis.event_type import EventType
+from nvflare.apis.fl_context import FLContext
+from nvflare.apis.shareable import Shareable
+from nvflare.app_common.abstract.params_converter import ParamsConverter
+
+
+def _get_params_converter(params_converter_id: str, fl_ctx: FLContext) -> ParamsConverter:
+    c = fl_ctx.get_engine().get_component(params_converter_id)
+
+    if c:
```

Comment:
What if "c" is None? I think we should simply check "if not isinstance(c, ParamsConverter)" and then raise exception.